cmd/blocking-issue-creator/main.go

```diff
@@ -103,9 +104,10 @@ func main() {
 
 		if err := manageIssues(client, botUser.Login, repoInfo, branches, logger); err != nil {
 			failed = true
-			return nil
 		}
 
+		//sleep to respect Github's API 30 requests per minute limit, absolute minimum is above 2 seconds
+		time.Sleep(5 * time.Second)
```

Comment:
I went with an arbitrary, fixed time setting which I believe is safe (see the description).
I think it's fine, but if your practice points to configurable param, it's fine too. I can change.